release/release_guide.md

```diff
@@ -18,9 +18,21 @@
 Kyuubi Docker Release Guide
 ===
 
+## Prerequisites
+
+Make sure Kyuubi has released a new version.
+
 ## Steps
 
-1. Make sure Kyuubi has released a new version
-2. Submit a pull request to update the `release_version` file to the new version
-3. After pull request merged. Push a new tag.
-4. After new tag created, github action will trigger a workflow to build and push new image to dockerhub
+### Minor release
```

Comment:
Let's call it "feature release"

https://kyuubi.readthedocs.io/en/master/contributing/code/release.html#cut-branch-if-for-feature-release